### PR TITLE
KAT-1895 add show/hide by default in password field [2.0.7 - patch]

### PR DESCRIPTION
**Pull request by tato123**
- Adds a show / hide option by default for inputs of type `password`
- Developers can disable show/hide by using flag `disableShowPassword`
- Updated docs

### CODE REVIEW

File: src/components/Input/Input.js [hunk 1]
```diff
@@ -138,7 +153,33 @@ class Input extends React.Component {
...
+  renderPasswordField = () => {
+    const toggleState = () => this.state._type === 'password' ? 'text' : 'password';
+    const handleClick = (evt) => {
+      evt.preventDefault();
+      this.setState({
+        _type: toggleState(this.state._type)
+      });
+    }
+
+    const getText = () => {
```
Comment (a-type) [inline src/components/Input/Input.js]: This function name seems kind of ambiguous for an input.

File: src/components/Input/Input.js [hunk 2]
```diff
@@ -138,7 +153,33 @@ class Input extends React.Component {
...
+      return this.state._type === 'password' ? 'Show' : 'Hide'
+    }
+
+    return (
+      <div style={{position:'relative'}}>
+          <div style={{position:'absolute', right: '10px', top: '25%'}}>
```
Comment (a-type) [inline src/components/Input/Input.js]: Didn't feel like using styled-components for these? I can certainly see why, just curious.

File: src/components/Input/Input.js (hunk 1 above)
Comment (tato123) [reply src/components/Input/Input.js]: Yup, you're right, that was just laziness on my part. Fixed

File: src/components/Input/Input.js (hunk 2 above)
Comment (tato123) [reply src/components/Input/Input.js]: This was honestly just carelessness on my part leaving this in while I played around with it. Changed to styled-components

**Pull request merged**